src/apollo/Apollo.ts

```diff
@@ -283,13 +292,19 @@ export default class Apollo implements ApolloInterface {
         }
 
         const seed = Buffer.from(seedStr, "hex");
-        const derivationPath = DerivationPath.fromPath(
-          Buffer.from(derivationPathStr, "hex").toString(),
-        );
 
-        const extendedKey = KeyDerivation.deriveKey(seed, derivationPath);
+        const newExtendedKey = HDKey.InitFromSeed(
+          Int8Array.from(seed),
+          0,
+          BigIntegerWrapper.initFromInt(0)
```

Comment:
not an issue, but I thought we were intending to hide bigInteger from the consumer, something for next iteration?